app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -25,8 +27,8 @@ const NewServiceForm = (props: Props) => {
   }
 
   const formToRender = () => formMode === 'manual' || !isServiceDiscoveryAccessible
-    ? <ServiceManualForm formActionPath={adminServicesPath}/>
-    : <ServiceDiscoveryForm formActionPath={providerAdminServiceDiscoveryServicesPath} setLoadingProjects={setLoadingProjects} />
+    ? <ServiceManualForm formActionPath={adminServicesPath} apiap={apiap} backendApis={backendApis} />
```

Comment:
No need to propagate `apiap` and `backendApis` down. Since the form doesn't depend on the `formMode`. Simply add the new field after `formToRender()`.

Reply:
True, you are passing `apiap` and `backendApis` to **ServiceManualForm** -> **ServiceManualListItems** then rendering with `{apiap && <SelectBackendApi backendApis={backendApis} />}`

And doing the same for **ServiceDiscoveryForm** -> **ServiceDiscoveryListItems** then `{apiap && <SelectBackendApi backendApis={backendApis} />}`

But it's unnecessary, as you are duplicating the same logic in 2 components. You can simply add **SelectBackendApi** here